content/docs/start/data-and-model-access.md

```diff
@@ -2,29 +2,29 @@
 title: 'Get Started: Data and Model Access'
 ---
 
-# Get Started: Data and Model Access
+# Data and Model Access
```

Comment:
These are by design for SEO.

Reply:
Are you sure?

- "Get Started" already appears in the `head.title` tag
- it also appears in the containing `nav` section
- does it really need to be in the `body.h1` element too?
- other subsections in the `nav` don't prefix their supersections in their `body.h1`s

is "Get Started" is some sort of SEO exception to the rule?

Reply:
> "Get Started" already appears in the head.title tag

Yep, I think that was the main reason: to match the HTML title (I guess helps ranking). Page previews on Google or anywhere will not always have the "Get Started" prefix, which can make it hard to clearly differentiate GS content vs. other similarly titled docs like the Versioning use case, Experiments guide, or Pipelines file ref. We want users to click on and land on the most appropriate content (decreases bounce rate).

That's what I remember but it was a while ago we made this decision. In any case if there's no strong reason to change this lets trust our previous choices, I think.

Reply:
> other subsections in the nav don't prefix their supersections in their body.h1s

The one tutorial we have does. And all the How to pages.

Reply:
Hmm may have to re-look at this later. If it's a thing, we could automate it rather than duplicate the title in all source files.

Reply:
Yep, let's roll back and make an issue to look into it?

Reply:
yes; reverted (forgot to push earlier).